examples/nlp/question-answering.py

```diff
@@ -0,0 +1,427 @@
+"""
+Title: Question Answering with Hugging Face Transformers
+Author: Matthew Carrigan and Merve Noyan
+Date created: 13/01/2022
+Last modified: 13/01/2022
+Description: Question answering implementation using Keras and Hugging Face Transformers.
+"""
+"""shell
```

Comment:
Please add an "introduction" section for the problem description and the installation instructions

Reply:
Added in my last commit 🙌🏼